app/assets/stylesheets/modules/slick.scss

```diff
@@ -0,0 +1,109 @@
+.slick-slider {
+  position: relative;
```

Comment:
Properties should be ordered -moz-box-sizing, box-sizing, display, position, -webkit-tap-highlight-color, -ms-touch-action, touch-action, -webkit-touch-callout, -khtml-user-select, -moz-user-select, -ms-user-select, -webkit-user-select, user-select